components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -462,6 +462,18 @@ def relationship=(value)
       application.ensure_relationship_with_primary(self, value)
     end
 
+    def ethnicity
+      # Retrieve the raw 'ethnicity' value from the database
+      # Return an empty array if 'ethnicity' is nil, ensuring consistent array behavior
+      read_attribute(:ethnicity) || []
+    end
+
+    def tribe_codes
+      # Retrieve the raw 'tribe_codes' value from the database
+      # Return an empty array if 'tribe_codes' is nil, ensuring consistent array behavior
+      read_attribute(:tribe_codes) || []
+    end
+
```

Comment:
Same here. Don't we want to add defaults for the fields `:ethnicity` and `:tribe_codes`?